src/extractFromJS.ts

```diff
@@ -0,0 +1,21 @@
+export function findTaggedTemplateLiteralsInJS(jsCode: string, tag: string): string[] {
+  // m for multiline
+  // g for matching multiple results
+  // capture the text inside the template literal with parentheses
+  const regex = new RegExp(tag + '\`([\\s\\S]+?)\`', 'mg');
```

Comment:
If we wanted to be really correct about it we would probably need to parse the AST, but I agree that it would be easy enough to just support the specific case where there is a space.